SofaKernel/modules/SofaSimpleFem/src/SofaSimpleFem/TetrahedronFEMForceField.inl

```diff
@@ -1801,19 +1801,22 @@ void TetrahedronFEMForceField<DataTypes>::draw(const core::visual::VisualParams*
 
     if (_showVonMisesStressPerNode.getValue())
     {
-        if (!_computeVonMisesStress.getValue())
+        if (_computeVonMisesStress.getValue()<=0)
         {
             msg_warning() << "Von Mises Stress Per Node can only be displayed if option computeVonMisesStress is set to true";
```

Comment:
"computeVonMisesStress is set to true": `computeVonMisesStress` is not a boolean. I suggest a clearer message, not implying a boolean variable.
Also, what happens when `computeVonMisesStress > 2`? What do you think about adding a `parse` function to verify the Data?

Reply:
I would even ask why ```computeVonMisesStress``` is not a boolean? If it is an option as @alxbilger said, a better message and even renaming the Data to ```VonMisesStressOption```
And I would say that at init (even better in a callback) of the component showVonMisesStressPerNode should set computeVonMisesStress to true.

Reply:
///< compute and display von Mises stress: 0: no computations, 1: using corotational strain, 2: using full Green strain

Reply:
I agree for a check of `computeVonMisesStress` value
I agree for changing the name in another PR